spark-doris-connector/spark-doris-connector-base/src/main/java/org/apache/doris/spark/config/DorisConfig.java

```diff
@@ -86,8 +86,10 @@ private void checkOptions(Map<String, String> options) throws OptionRequiredExce
             String feNodes = options.get(DorisOptions.DORIS_FENODES.getName());
             if (feNodes.isEmpty()) {
                 throw new IllegalArgumentException("option [" + DorisOptions.DORIS_FENODES.getName() + "] is empty");
-            } else if (!feNodes.contains(":")) {
-                throw new IllegalArgumentException("option [" + DorisOptions.DORIS_FENODES.getName() + "] is not in correct format, for example: host:port[,host2:port]");
+            } else if (feNodes.contains(":")) {
+                if (!feNodes.matches("([\\w-.]+:\\d+,)*([\\w-.]+:\\d+)"))  {
```

Comment:
Can you add a test for this regex?

Reply:
add test case and update this pr for some bugs fix